BookingsApi/BookingsApi.DAL/Commands/JudiciaryPersonCommandBase.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace BookingsApi.DAL.Commands
+{
+    public abstract class JudiciaryPersonCommandBase
```

Comment:
this base class should be implementing `ICommand` so that it doesn't need to be specified in both inherited classes

Reply:
Done